### PR TITLE
Add firefox/home.ftl

### PR DESCRIPTION
**Pull request by alexgibson**
Bedrock pull request: https://github.com/mozilla/bedrock/pull/8974

### CODE REVIEW

File: en/firefox/home.ftl [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+firefox-home-every-single-firefox = Every single { -brand-name-firefox } product honors our Personal Data Promise: <strong>Take less. Keep it safe. No secrets.</strong>
+firefox-home-share-large-files-without = Share large files without prying eyes
+firefox-home-start-sending-files = Start sending files safely
+firefox-home-trade-clickbait-for = Trade clickbait for quality content
+firefox-home-learn-more-about-pocket = Learn more about { -brand-name-pocket }
+
+# The strong tags around "privacy" add a special underline. The underline breaks if it is on two words, so please omit the strong tags if they need to be around multiple words in your language.
+firefox-home-one-login-all-your = One login. All your devices. A family of products that respect your <strong>privacy</strong>.
+
+firefox-home-join-firefox = Join { -brand-name-firefox }
```
Comment (alexgibson) [inline en/firefox/home.ftl:29]: @peiying2 please note there's one new string here which is currently [untranslated in production](https://www.mozilla.org/de/firefox/). I believe this bug is due to the page relying on a .lang string that was removed from the L10n-repo pre-swtichover.

I've added it back here, so locales can get it translated.

**Comment by peiying2** [comment]
Looks good. I do have a question. After the newly added string, where can I find the strings in the remainder of the file? Or does it come from a different page?

**Comment by alexgibson** [comment]
> After the newly added string, where can I find the strings in the remainder of the file? Or does it come from a different page?

I'm not sure I quite understand your question?

File: en/firefox/home.ftl [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+firefox-home-learn-more-about-joining = Learn more about joining { -brand-name-firefox }
+firefox-home-get-the-browser-extension = Get the browser extension
```
Comment (peiying2) [inline en/firefox/home.ftl:31]: Where can I find the strings from here to the end of the file? I can't find them on the page

Comment (alexgibson) [reply en/firefox/home.ftl:31]: Most of these strings are replaced by images (Firefox Browser) logo etc. They are still in the page, and are still important for a11y and SEO. Would a group comment be useful here to help give context?

**Comment by alexgibson** [comment]
@peiying2 updated, thanks

**Pull request merged**